contracts/party/PartyGovernance.sol

```diff
@@ -136,6 +138,10 @@ abstract contract PartyGovernance is
         uint96 votes; // -1 == vetoed
         // Number of total voting power at time proposal created.
         uint96 totalVotingPower;
+        /// @notice Number of hosts at time proposal created
+        uint8 proposalTimeNumHosts;
```

Comment:
Hosts can abdicate their host status at any point which decreases the number of hosts in the party. Therefore, we need to checkpoint the hosts at proposal time to avoid manipulation.

Reply:
> Hosts can abdicate their host status at any point which decreases the number of hosts in the party. Therefore, we need to checkpoint the hosts at proposal time to avoid manipulation.

I know, this is purely a style thing that you can take or leave. I'm just saying you could rename the variable from `proposalTimeNumHosts` to `numHosts`. This is following the way `totalVotingPower` is named in the struct above `numHosts`.

Reply:
Oh got it, yeah we can do that